stable/atlantis/README.md

```diff
@@ -31,13 +31,16 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
 | Parameter                                   | Description                                                                                                                                                                                                                                                                                                                                                | Default                |
 | ------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------- |
 | `allow_repo_config` | Whether to allow the use of [atlantis.yaml files](https://www.runatlantis.io/docs/atlantis-yaml-reference.html). | `false`                  |
+| `annotations` | Specifies additional annotations to use for the StatefulSet | n/a                  |
```

Comment:
Any possibility of a foreseeable conflict with this name?  (i.e. Anyplace else that might need annotations?) I know that preventing breaking changes in a Helm chart is a fool's game, but wanted to ensure we did our due diligence.